ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml

```diff
@@ -6,6 +6,9 @@ ref:
     requests:
       cpu: 1000m
       memory: 400Mi
+  dependencies:
+    - env: WMCO_IMAGE
```

Comment:
In the commit message you mentioned
>variables needed by the WMCO e2e test script.

for my own education, where the `WMCO_IMAGE`  variable is used?

Reply:
It will be used in the e2e hack script instead of the [IMAGE_FORMAT ](https://github.com/openshift/windows-machine-config-operator/blob/6901ac3ed2891b7627a1431ab66703358b902671/hack/run-ci-e2e-test.sh#L95)variable.
In fact. looking at it again I think the name should be set to windows-machine-config-operator-test since that's what it is replaced with in the line linked above.